src/App.tsx

```diff
@@ -152,8 +178,27 @@ export default function App() {
             Calculate
           </Button>
         </Grid>
-        <h2>Results:</h2>
-        {results && <ReactJson src={results} enableClipboard={true} theme="shapeshifter:inverted" collapsed={2} />}
+        <Grid container>
+          <Grid container item xs={6} direction="row">
+            <div>
+              <h2>Results:</h2>
+              {results && (
+                <ReactJson src={results} enableClipboard={true} theme="shapeshifter:inverted" collapsed={2} />
+              )}
+            </div>
+          </Grid>
+          <Grid container item xs={6} direction="row" wrap="nowrap">
+            {htmls &&
+              htmls.map(html => {
+                return (
+                  <div key={html.groupId}>
```

Comment:
```suggestion
                  <div key={html.groupId} className={classes.highlightedMarkup}>
```

See comment above for the contents of this class